features/features/package_managers/rebar_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'feature_helper'
+
+describe "Rebar Dependencies" do
+  # As a Node developer
+  # I want to be able to manage NPM dependencies
```

Comment:
Should be "As an Erlang developer / I want to be able to manage rebar dependencies".